### PR TITLE
Upgrade to AKS version 1.24.0 and change gateway subnet from /28 to /24

### PR DESCRIPTION
**Pull request by skabou**
Upgrade to AKS version 1.24.0
Change gateway subnet from /28 to /24

Fixes: #51

### CODE REVIEW

File: cluster-stamp.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
             }
         },
         "kubernetesVersion": {
-            "defaultValue": "1.22.11",
```
Comment (ckittel) [inline cluster-stamp.json:96]: Please update the README.md file (search for `1.22`)

File: networking/topology.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ In the future, this VNet might hold more subnets like [ACI Provider instance] su
...
-| Azure Application Gateway Subnet                       | 0            | [11]               | 0                | 0          | 0                        | 100           | 100                 | 0                     | 0                         | 5                                        | 0                   | 16                    | 16                 | 28                 | 10.240.4.0/28  | 10.240.4.16 | 10.240.4.31   |
+| Azure Application Gateway Subnet                       | 0            | [251]              | 0                | 0          | 0                        | 100           | 100                 | 0                     | 0                         | 5                                        | 0                   | 256                   | 256                | 24                 | 10.240.5.0/24  | 10.240.5.0  | 10.240.5.255  |
```
Comment (ckittel) [inline networking/topology.md:36]: That's max instances of App Gateway V2.  See related change below.

```suggestion
| Azure Application Gateway Subnet                       | 0            | [125]              | 0                | 0          | 0                        | 100           | 100                 | 0                     | 0                         | 5                                        | 0                   | 256                   | 256                | 24                 | 10.240.5.0/24  | 10.240.5.0  | 10.240.5.255  |
```

File: networking/topology.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ In the future, this VNet might hold more subnets like [ACI Provider instance] su
 3. [Private Endpoints] Private Links are created for ACR and Azure KeyVault, so these Azure services can be accessed using Private Endpoints within the Spoke vNet, specifically allocating an private Ip address from the AKS System and User Nodepool Subnet.
 
 [27<sup>1</sup>]: https://docs.microsoft.com/azure/vpn-gateway/vpn-gateway-about-vpn-gateway-settings#gwsub
-[11]: https://docs.microsoft.com/azure/application-gateway/configuration-overview#size-of-the-subnet
+[251]: https://docs.microsoft.com/azure/application-gateway/configuration-overview#size-of-the-subnet
```
Comment (ckittel) [inline networking/topology.md:48]: old link didn't work.

```suggestion
[125]: https://docs.microsoft.com/azure/application-gateway/configuration-infrastructure#size-of-the-subnet
```

**Review by ckittel — CHANGES_REQUESTED** [review]
Just a couple small things

File: cluster-stamp.json (hunk 1 above)
Comment (skabou) [reply cluster-stamp.json:96]: Nice catch, updated.

**Review by ckittel — APPROVED** [review]
LGTM 🎸

**Pull request merged**